{{cookiecutter.project_dashed}}-src/{{cookiecutter.project_namespace}}-config-example.py

```diff
@@ -17,12 +17,16 @@ class DevProfile(object):
 
     MAIL_DEFAULT_SENDER = '{{cookiecutter.developer_email}}'
 
+    KEG_LOG_SYSLOG_ENABLED = False
+
     # Needed by at least KegAuth for sending emails from the command line
     SERVER_NAME = 'localhost:5000'
 
     CELERY = {
```

Comment:
We have a config function but we aren't using it:

https://github.com/level12/keg-app-cookiecutter/blob/master/%7B%7Bcookiecutter.project_dashed%7D%7D-src/%7B%7Bcookiecutter.project_namespace%7D%7D/celery/config.py